src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace Microsoft.Graph
 {
+    using Microsoft.Kiota.Abstractions;
```

Comment:
I'm not sure why we still need base request since the generated fluent API is not going to rely on it anymore?